platform/base.go

```diff
@@ -76,25 +75,31 @@ func (bc *BaseCluster) PasswordSSHClient(ip string, user string, password string
 	return sshClient, nil
 }
 
-func (bc *BaseCluster) SSH(m Machine, cmd string) ([]byte, error) {
+func (bc *BaseCluster) SSH(m Machine, cmd string) (stdout, stderr []byte, err error) {
```

Comment:
This should be a new function to avoid the temptation to discard stderr when test writers don't particularly care about it.